geniza/corpus/dates.py

```diff
@@ -183,15 +183,15 @@ def original_date(self):
             [self.doc_date_original, self.get_doc_date_calendar_display()]
         ).strip()
 
-    @property
-    def standard_date(self):
+    @staticmethod
+    def standard_date_display(standard_date):
```

Comment:
Would this be better as a standalone utility method? it feels a bit odd attached to the document / document mixin now that you're also using it elsewhere.  (maybe slightly simpler as standalone instead of static, IDK)